components/cache/adapters/redis_adapter.rst

```diff
@@ -0,0 +1,127 @@
+.. index::
+    single: Cache Pool
+    single: Redis Cache
+
+Redis Cache Adapter
+===================
+
+This adapter stores the values in-memory using  one (or more) `Redis server`_ instances.
+Unlike the ACPu adapter, and similarly to the Memcached adapter, it is not limited to the current
+server's shared memory; you can store contents independent of your PHP environment.
+The ability to utilize a cluster of servers to provide redundancy and/or fail-over is also
+available.
+
+.. caution::
+
+    **Requirements:** At least one `Redis server`_ must be installed and running to use this
+    adapter. Additionally, this adapter requires a compatible extension or library that implements
+    ``\Redis``, ``\RedisArray``, ``RedisCluster``, or ``\Predis``.
+
+
+This adapter expects a `Redis`_, `RedisArray`_, `RedisCluster`_, or `Predis`_ instance to be
+passed as the first parameter. A namespace and default cache lifetime can optionally be passed
+as the second and third parameters::
+
+    use Symfony\Component\Cache\Adapter\RedisAdapter;
+
+    $cache = new RedisAdapter(
+
+        // the object that stores a valid connection to your Redis system
+        \Redis $redisConnection,
+
+        // the string prefixed to the keys of the items stored in this cache
+        $namespace = '',
+
+        // the default lifetime (in seconds) for cache items that do not define their
+        // own lifetime, with a value 0 causing items to be stored indefinitely (i.e.
+        // until MemcachedAdapter::clear() is invoked or the server(s) are purged)
+        $defaultLifetime = 0
+    );
+
+Configure the Connection
+------------------------
+
+The :method:`Symfony\\Component\\Cache\\Adapter\\RedisAdapter::createConnection`
+helper method allows creating and configuring the Redis client class instance using a
+`Data Source Name (DSN)`_ or an array of DSNs::
+
+    use Symfony\Component\Cache\Adapter\RedisAdapter;
+
+    // pass a single DSN string to register a single server with the client
+    $client = RedisAdapter::createConnection(
+        'redis://localhost'
+    );
+
+    // pass an array of DSN strings to register multiple servers with the client
+    $client = RedisAdapter::createConnection(array(
```

Comment:
Do we want to get a PR together for it in the immediate so as to offer the same ability for both MemcachedAdapter and RedisAdapter? If you don't have the time, I can look to getting a PR together? Or do you not care one way or the other? How much time do we have before the 3.3 release (when these docs need to be live)?